fdbserver/BlobMigrator.actor.cpp

```diff
@@ -259,9 +261,13 @@ class BlobMigrator : public NonCopyable, public ReferenceCounted<BlobMigrator> {
 
 				// Count incompleted size
 				int64_t incompleted = 0;
-				for (auto i = 0; i < ranges.size() - 1; ++i) {
+				for (auto i = 0; i < ranges.size(); ++i) {
 					if (ranges[i].value == serverKeysTrue) {
-						KeyRangeRef range(ranges[i].key, ranges[i + 1].key);
+						KeyRef end = normalKeys.end;
```

Comment:
an edge case that the last element in ranges is not \xff

Reply:
When can this happen in this test? It's not using tenants or anything

Reply:
discussed offline. it is used to check key ranges the blob migrator owns.